content/manuals/subscription/change.md

```diff
@@ -0,0 +1,114 @@
+---
+description: Learn how to change your Docker subscription
+keywords: Docker Hub, upgrade, downgrade, subscription, Pro, Team, business, pricing plan
+title: Change your subscription
+aliases:
+- /docker-hub/upgrade/
+- /docker-hub/billing/upgrade/
+- /subscription/upgrade/
+- /subscription/downgrade/
+- /subscription/core-subscription/upgrade/
+- /subscription/core-subscription/downgrade/
+- /docker-hub/cancel-downgrade/
+- /docker-hub/billing/downgrade/
+weight: 30
+---
+
+{{< include "tax-compliance.md" >}}
+
+The following sections describe how to change plans when you have a Docker
+subscription plan or legacy Docker subscription plan.
+
+> [!NOTE]
+>
+> Legacy Docker plans apply to Docker subscribers who last purchased or renewed
+> their subscription before December 10, 2024. These subscribers will keep
+> their current plan and pricing until their next renewal date that falls on or
+> after December 10, 2024. To see purchase or renewal history, view your
+> [billing history](../billing/history.md). For more details about legacy
+> subscriptions, see [Announcing Upgraded Docker
+> Plans](https://www.docker.com/blog/november-2024-updated-plans-announcement/).
+
+## Upgrade your subscription
+
+When you upgrade a Docker plan, you immediately have access to all the features and entitlements available in your Docker subscription plan. For detailed information on features available in each subscription, see [Docker Pricing](https://www.docker.com/pricing).
+
+{{< tabs >}}
+{{< tab name="Docker plan" >}}
+
+To upgrade your Docker subscription:
+
+1. Sign in to [Docker Home](https://app.docker.com/).
+2. Under Settings and administration, select **Billing**.
+3. Optional. If you're upgrading from a free Personal plan to a Team plan and want to keep your username, [convert your user account into an organization](../admin/convert-account.md).
+4. Select the account you want to upgrade in the drop-down at the top-left of the page.
+5. Select **Upgrade**.
+6. Follow the on-screen instructions to complete your upgrade.
+
+{{< /tab >}}
+{{< tab name="Legacy Docker plan" >}}
+
+You can upgrade a legacy Docker Core, Docker Build Cloud, or Docker Scout subscription plan to a Docker subscription plan that includes access to all tools.
```

Comment:
Is this intentional that we're saying "Docker Core" here, so that it differentiates from the other products?
Or should it say "Legacy Docker plan"?